scenario/mocking.py

```diff
@@ -368,10 +362,8 @@ def secret_add(
     ) -> str:
         from scenario.state import Secret
 
-        secret_id = self._generate_secret_id()
         secret = Secret(
-            id=secret_id,
-            contents={0: content},
+            latest=content,
```

Comment:
and this is why I'd rather more explicit argument names :) 

`Secret(content=content` reads so much better.